<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="hp" failures="1" tests="2" name="Default test(failed)" time="0.044" errors="0" timestamp="2022-11-20T16:53:17 IST">
  <testcase classname="testngpac.FailedTestCaseRerun" name="tc1" time="0.006"/>
  <testcase classname="testngpac.FailedTestCaseRerun" name="tc4" time="0.001">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testngpac.FailedTestCaseRerun.tc4(FailedTestCaseRerun.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- tc4 -->
  <testcase classname="testngpac.FailedTestCaseRerun" name="tc3" time="0.0">
    <ignored/>
  </testcase> <!-- tc3 -->
  <testcase classname="testngpac.FailedTestCaseRerun" name="tc2" time="0.0">
    <ignored/>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Default test(failed) -->
